src/components/standalone/multi-wan/RuleCreator.vue

```diff
@@ -34,6 +41,7 @@ const {
   sourcePort,
   destinationAddress,
   destinationPort,
+  sticky_toggle,
```

Comment:
Let's use the same style of other variables: `stickyToggle` or, probably better, simply `sticky`